src/contexts/swr/useFetcher.ts

```diff
@@ -39,7 +39,8 @@ export const _fetch: FetcherType = async (path, action, parameters = {}) => {
     credentials: 'include',
   };
 
-  const localJwt = window.localStorage.getItem(JWT_LOCAL_STORAGE_KEY);
+  const localJwt =
+    typeof window === 'undefined' ? null : window.localStorage.getItem(JWT_LOCAL_STORAGE_KEY);
```

Comment:
Had to change this so we can make requests on the server. Hopefully this stuff can go away now that we have Cookie auth